benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -15,6 +15,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "generatedDOB", "Label": "Appellant Date of Birth", "FieldType": "Date", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "regionalProcessingCenter", "Label": "RegionalProcessingCenter", "FieldType": "regionalProcessingCenter", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "sscsDocument", "Label": "Case documents", "FieldType": "Collection", "FieldTypeParameter": "sscsDocument", "RegularExpression": "No", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "addedDocuments", "Label": "Added documents", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
```

Comment:
You need a CRUD permission for this new field in AuthorisationCaseField.json tab